test_plans/header-file.asciidoc

```diff
@@ -0,0 +1,44 @@
+:sectnums:
+:xrefstyle: short
+
+= Test plan for header files and namespaces
+
+This is a test plan for the header files and namespaces as described in SYCL 2020 Section 4.3.
+
+Estimated development time is one day.
+
+== Testing scope
+
+=== Backend coverage
+
+All the tests described below are performed for any SYCL backend.
+
+=== Device coverage
+
+All the tests described below are performed only on the default device that is selected on the CTS command line.
+
+== Tests
+
+The following combinations for namespaces and header files exist:
+
+|===
+|    |`::sycl` |`::CL::sycl`
```

Comment:
```suggestion
|    |`::sycl` |`::cl::sycl`
```

Both SYCL 2020 and SYCL 1.2.1 say the `cl` namespace is lowercase.

Reply:
That is correct, I have fixed it.